src/openrct2/ride/Ride.cpp

```diff
@@ -3918,7 +3918,14 @@ static int32_t ride_check_block_brakes(CoordsXYE* input, CoordsXYE* output)
  */
 static bool ride_check_track_contains_inversions(CoordsXYE* input, CoordsXYE* output)
 {
-    ride_id_t rideIndex = input->element->AsTrack()->GetRideIndex();
+    if (input->element == nullptr)
+        return false;
+
+    const auto& trackElement = input->element->AsTrack();
+    if (trackElement == nullptr)
```

Comment:
This can't be a `nullptr`, consistering the `input->element == nullptr` above, right?

Reply:
It can most definitely be a nullptr. AsTrack() will return nullptr if the element is not a track, but another tile element type.